.circleci/config.yml

```diff
@@ -119,7 +111,7 @@ jobs:
       - run:
           name: test
           command: |
-            pytest src packages/*/test* packages/micropip/micropip/ pyodide_build -v -k chrome
+            pytest src packages/*/test* pyodide_build -v -k chrome
```

Comment:
There idea there was to run micropip doctests, but you are right that there aren't any at the moment.